cmd/createorupdate/createorupdate.go

```diff
@@ -69,7 +69,7 @@ func validate() error {
 	return nil
 }
 
-func delete(ctx context.Context, log *logrus.Entry, rpc sdk.OpenShiftManagedClustersClient, resourceGroup string, noWait bool) error {
+func delete(ctx context.Context, log *logrus.Entry, rpc v20180930preview.OpenShiftManagedClustersClient, resourceGroup string, noWait bool) error {
```

Comment:
In a follow-up I am planning on introducing an interface here that will be backed by both the external customer and admin types as implementations.